src/block/precomputed.rs

```diff
@@ -15,12 +17,14 @@ pub struct BlockLogContents {
     pub(crate) state_hash: String,
     pub(crate) blockchain_length: Option<u32>,
     pub(crate) contents: Vec<u8>,
+    pub(crate) global_slot_since_genesis: GlobalSlotNumberV1,
 }
 
 #[derive(Debug, Deserialize, Serialize)]
 pub struct BlockLog {
     scheduled_time: String,
     protocol_state: ProtocolStateJson,
+    consensus_state: ConsensusStateJson,
```

Comment:
We don't need this extra field. The `consensus_state` object lives inside the `body` object inside the `protocol_state` object. The *beautified logs* are a lot easier to read to confirm this.